lib/ansible/modules/identity/ipa/ipa_dnszone.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Fran Fitzpatrick (francis.x.fitzpatrick@gmail.com)
+# Borrowed heavily from other modules by Abhijeet Kasurde (akasurde@redhat.com)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
```

Comment:
Can you make this as single liner GPL licence like [this](https://github.com/ansible/ansible/blob/513b582ba39a83c9400b986ff02fdd43c9ed0820/lib/ansible/modules/identity/ipa/ipa_dnsrecord.py#L4)? This is newly added thing.